packages/ember-glimmer/lib/components/link-to.ts

```diff
@@ -534,10 +518,14 @@ const LinkComponent = EmberComponent.extend({
   */
   disabled: computed({
     get(_key: string): boolean {
+      // always returns false for `get` because (due to the `set` just below)
+      // the cached return value from the set will prevent this getter from _ever_
+      // being called after a set has occured
       return false;
     },
+
     set(_key: string, value: any): boolean {
-      if (value !== undefined) { this.set('_isDisabled', value); }
+      this._isDisabled = value;
```

Comment:
The changes here some right to me in general, but they do change the semantics a bit, don't they (the truthy vs. not undefined)? Could they break something for somebody?

Reply:
@simonihmig if you take a look at where `this._isDisabled` is actually checked (in `_invoke`), we have been doing a truthy check there for a while so changing this to always update it just makes it simpler...